data/tlg2200/tlg001/__cts__.xml

```diff
@@ -1,7 +1,7 @@
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg2200" xml:lang="grc" urn="urn:cts:greekLit:tlg2200.tlg001">
   <ti:title xml:lang="lat">Epistulae 1-1544</ti:title>
```

Comment:
@AlisonBabeu 
I don't understand why the title would be 1-1544 when the label is 1-839. (plus I see only 831 letters in here).
Wouldn't the title be better presented as `<ti:title xml:lang="lat">Epistulae</ti:title>`?

Reply:
I think I left that there because that's the total number of letters and wanted to remind myself that all of them weren't in the file.  Epistulae would be a much better title.